release notes/v0.51.0.md

```diff
@@ -9,6 +9,46 @@ k6 `v0.51.0` is here 🎉! This release includes:
 - `#pr`, `<small_break_1>`
 - `#pr`, `<small_break_2>`
 
+### Browser `Tap` is now an async method grafana/xk6-browser#1268
+
+This release converts the `Tap` method in the `browser` module into an asynchronous method. This change is necessary to ensure that the method can be used in async contexts and to align with the rest of the browser module's planned asynchronous API. To use the `Tap` method, you must now add the `await` keyword before the method call.
+
+Affected components:
+- [`locator.tap`](https://grafana.com/docs/k6/latest/javascript-api/k6-experimental/browser/locator/tap/)
+- [`page.tap`](https://grafana.com/docs/k6/latest/javascript-api/k6-experimental/browser/page/tap/)
+- [`frame.tap`](https://grafana.com/docs/k6/latest/javascript-api/k6-experimental/browser/frame/)
+- [`elementHandle.tap`](https://grafana.com/docs/k6/latest/javascript-api/k6-experimental/browser/elementhandle/)
```

Comment:
I'll update these links once I update the documentation. I'm doing my best, but let's be ready since k6 wants to release this week.